ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -0,0 +1,52 @@
+base_images:
+  cli:
+    cluster: https://api.ci.openshift.org
+    name: "4.5"
+    namespace: ocp
+    tag: cli
+binary_build_commands: make build
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+images:
+- dockerfile_path: openshift-ci/build-root/Dockerfile
+  from: bin
+  inputs:
+    cli:
+      paths:
+      - destination_dir: .
+        source_path: /usr/bin/oc
+  to: oc-bin-image
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  cluster: https://api.ci.openshift.org
+  name: "4.5"
+  namespace: ocp
+test_binary_build_commands: make test
+tests:
+- as: e2e-aws
+  steps:
+    cluster_profile: aws
+    test:
+    - as: e2e-test
+      commands: ./bin/extended-platform-tests run openshift/conformance/parallel --run
+        "OLM should be installed with"
+      from: oc-bin-image
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    workflow: ipi-aws
```

Comment:
@petr-muller Do you know how to select the OCP version? Thanks! I guess this failure related to the OCP version.

Reply:
Please read the docs: https://ci-operator-configresolver-ui-ci.apps.ci.l2s4.p1.openshiftapps.com/help/ci-operator#release

In this case the OCP version was a most current 4.5 (heads of branches), like specified in your `tag_specification`.

Reply:
Thanks! Yes, but, it works well in my local OCP 4.5, I could not find the root cause of this failure.